### PR TITLE
Use idiomatic Kotlin in REST Assured tests

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: integration-tests/kotlin-serialization/src/test/kotlin/io/quarkus/it/kotser/ResourceTest.kt [hunk 1]
```diff
@@ -1,73 +1,80 @@
 package io.quarkus.it.kotser
 
 import io.quarkus.test.junit.QuarkusTest
-import io.restassured.RestAssured.given
+import io.restassured.RestAssured.*
```
Comment (gsmet) [inline integration-tests/kotlin-serialization/src/test/kotlin/io/quarkus/it/kotser/ResourceTest.kt:4]: Not sure it's a change you wanted to do?

Comment (geoand) [reply integration-tests/kotlin-serialization/src/test/kotlin/io/quarkus/it/kotser/ResourceTest.kt]: Good catch! Fixed

**Review by gsmet — APPROVED** [review]
For me less Kotlin is better Kotlin but YMMV :).

**Comment by geoand** [comment]
I'm also sceptical about various Koltin features, but I think this little RestAssured DSL improves things

**Comment by gsmet** [comment]
Backported to avoid conflicts.

**Comment by geoand** [comment]
Good idea

**Pull request merged**